### PR TITLE
11.0 pos menu port

### PR DESCRIPTION
**Pull request by fedoranvar**

### CODE REVIEW

**Review by Ramil-Mukhametzyanov — CHANGES_REQUESTED** [review]
There is no Available Product Sets option in POS settings.

**Review by Ramil-Mukhametzyanov — APPROVED** [review]

File: pos_menu/README.rst [hunk 1]
```diff
@@ -28,12 +29,12 @@ Maintainers
...
 
-HTML Description: https://apps.odoo.com/apps/modules/10.0/pos_menu/
+HTML Description: https://apps.odoo.com/apps/modules/11.0/pos_menu/
 
 Usage instructions: `<doc/index.rst>`_
 
 Changelog: `<doc/changelog.rst>`_
 
-Tested on Odoo 10.0 0351677a00df8bd76fa207a38dc2ecdefdc6940a
+Tested on Odoo 11.0 0351677a00df8bd76fa207a38dc2ecdefdc6940a
```
Comment (em230418) [inline pos_menu/README.rst]: This commit is from 12.0 branch
https://github.com/odoo/odoo/commit/0351677a00df8bd76fa207a38dc2ecdefdc6940a

File: pos_menu/README.rst [hunk 2]
```diff
@@ -9,7 +9,8 @@ Credits
 
 Contributors
 ------------
-* `Dinar Gabbasov <https://it-projects.info/team/GabbasovDinar>`__
+* `Gabbasov Dinar <https://it-projects.info/team/GabbasovDinar>`_
```
Comment (em230418) [inline pos_menu/README.rst]: We don't need to change two underscores to one

**Review by em230418 — CHANGES_REQUESTED** [review]
Everything works fine. Nonessential changes required in README.rst

File: pos_menu/README.rst (hunk 1 above)
Comment (fedoranvar) [reply pos_menu/README.rst]: changed commit

File: pos_menu/README.rst (hunk 2 above)
Comment (fedoranvar) [reply pos_menu/README.rst]: checked

File: pos_menu/README.rst [hunk 3]
```diff
@@ -28,12 +29,12 @@ Maintainers
...
 
-HTML Description: https://apps.odoo.com/apps/modules/10.0/pos_menu/
+HTML Description: https://apps.odoo.com/apps/modules/11.0/pos_menu/
 
 Usage instructions: `<doc/index.rst>`_
 
 Changelog: `<doc/changelog.rst>`_
 
-Tested on Odoo 10.0 0351677a00df8bd76fa207a38dc2ecdefdc6940a
+Tested on Odoo 11.0 e99c6ea835c119b478c853f7b6af511afe453c47
```
Comment (em230418) [inline pos_menu/README.rst]: Commit does not exist: https://github.com/odoo/odoo/commit/e99c6ea835c119b478c853f7b6af511afe453c47

Comment (fedoranvar) [reply pos_menu/README.rst]: how to insert id of commit? 

I'm doing 'git log -1'
putting in readme
git add . && git commit --amend

Am I right?

Comment (em230418) [reply pos_menu/README.rst]: Don't know. I do like this:

https://github.com/odoo/odoo/tree/11.0
Ctrl+F "Latest commit"
Copy hyperlink to the right
You will get
https://github.com/odoo/odoo/commit/<COMMIT_ID>

Comment (fedoranvar) [reply pos_menu/README.rst]: oh, I think I get it 

So we paste commit-id of odoo, not the module

Comment (GabbasovDinar) [reply pos_menu/README.rst]: git rev-parse HEAD

**Review by em230418 — APPROVED** [review]

**Comment by yelizariev** [comment]
I approve to merge it now

**Pull request merged**